integration-libs/epd-visualization/root/connectors/scene/storage.connector.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { NodesResponse } from './nodes-response';
+import { SceneAdapter } from './scene.adapter';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class StorageConnector {
```

Comment:
I cannot find this connector being used anywhere.

The services layer should depend on the connector layer (but not adapter). Only the connector should delegate work to a adapter.
For more, see examples in other libs or see https://sap.github.io/spartacus-docs/connecting-to-other-systems/